src/Intents/INSetProfileInCarIntent.cs

```diff
@@ -3,16 +3,27 @@
 using XamCore.Foundation;
 using XamCore.Intents;
 using XamCore.ObjCRuntime;
+using XamCore.UIKit;
 
 namespace XamCore.Intents {
 
 	public partial class INSetProfileInCarIntent {
 
+		[DesignatedInitializer]
 		public INSetProfileInCarIntent (NSNumber profileNumber, string profileLabel, bool? defaultProfile) :
 			this (profileNumber, profileLabel, defaultProfile.HasValue ? new NSNumber (defaultProfile.Value) : null)
 		{
 		}
 
+		protected INSetProfileInCarIntent (NSNumber profileNumber, string profileLabel, NSNumber defaultProfile)
+		{
+			// Apple created this change in 10,2
+			if (UIDevice.CurrentDevice.CheckSystemVersion (10, 2))
+				Handle = InitWithProfileNumberName (profileNumber, profileLabel, defaultProfile);
```

Comment:
Call `InitializeHandle` instead of setting the `Handle` property (since it will throw an exception if the handle is IntPtr.Zero).